### PR TITLE
Create website

### PR DESCRIPTION
**Pull request by liuba-larousse**
Just a page for putting all of my challenges at. :)

### References

Neon Sign
https://codemyui.com/flickering-neon-text-animation/

repeating-linear-gradient()
https://developer.mozilla.org/en-US/docs/Web/CSS/repeating-linear-gradient

Mix-blend-mode
https://developer.mozilla.org/en-US/docs/Web/CSS/mix-blend-mode

linear-gradient()
https://developer.mozilla.org/en-US/docs/Web/CSS/linear-gradient

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <div class="wrap">
+      <div class="header">
+        <h1>
+          <span>My p</span><span>roje</span><span>cts</span><span>!</span>
+        </h1>
+        <h2 class="bottom">If you can imagine it, you can create it</h2>
+      </div>
+      <div class="link-list">
+        <div class="column">
+          <a class="link" href="02_color_palette/index.html">Color Palette</a>
```
Comment (sandrina-p) [inline index.html:27]: This is a list of links, so you should be using `<ul>` :)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <link rel="stylesheet" href="css/style.css" />
+    <title>My Work</title>
+  </head>
+  <body>
+    <div class="wrap">
+      <div class="header">
+        <h1>
+          <span>My p</span><span>roje</span><span>cts</span><span>!</span>
+        </h1>
+        <h2 class="bottom">If you can imagine it, you can create it</h2>
```
Comment (sandrina-p) [inline index.html:23]: This is more like a paragraph, not a heading. Imagine this like a book, it does not make sense to be a h2 :)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Molle:ital@1&family=Rajdhani&family=Warnes&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>My Work</title>
+  </head>
+  <body>
+    <div class="wrap">
```
Comment (sandrina-p) [inline index.html:18]: Cool site eheh Maybe you want the image to be the size of the page, to prevent the page from scrolling down (not needed).

**Comment by sandrina-p** [comment]
in this PR you also staged files from other challenges. Remember: Only stage files that are related to this PR. All the other files should not even exist yet locally because they were not merged into master yet.

File: index.html (hunk 2 above)
Comment (liuba-larousse) [reply index.html:23]: Yes makes sence!

File: css/style.scss [hunk 4]
```diff
@@ -60,13 +60,21 @@ html {
...
 }
 
 .layer-three {
     width: 100%;
     height: 100%;
+<<<<<<< HEAD
     background: linear-gradient(135deg, rgba(255, 255, 255, 0) 65%, #43a6e3 65%);
+=======
+    background: linear-gradient(135deg, rgba(255, 255, 255, 0) 65%, #8756C7 65%);
+>>>>>>> 1db078439423f82da4a17ec0acee578fb455964c
```
Comment (sandrina-p) [inline css/style.scss:77]: @liuba-larousse, when you do a pull from another branch, sometimes there are conflicts, you should solve them before pushing the changes. [Here's a short tutorial about it](https://www.youtube.com/watch?v=lXPYqVjyqHM)

**Pull request merged**